specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json

```diff
@@ -8,14 +8,10 @@
   "paths": {
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AgFoodPlatform/farmBeats/{farmBeatsResourceName}/extensions/{extensionId}": {
       "put": {
-        "tags": [
-          "Extensions"
-        ],
+        "tags": ["Extensions"],
```

Comment:
Please avoid making superfluous format changes like this -- it just makes reviewing that much more difficult.

Reply:
Fixed this, ran prettier and it worked fine